tests/python2-libraries/requirements.txt

```diff
@@ -1,212 +1,207 @@
-simplejson
-setuptools
-six
-requests
-virtualenv
-pip
-distribute
-python-dateutil
-certifi
-boto
-pbr
-#wincertstore
-docutils
-pyasn1
-pyyaml
-jinja2
-markupsafe
-pytz
-nose
-lxml
-pycrypto
-rsa
-colorama
-botocore
-cffi
-awscli
-coverage
-jmespath
-pycparser
-pika
-django
-psycopg2
-paramiko
-ecdsa
-sqlalchemy
-mock
-redis
-werkzeug
-selenium
-bcdoc
-supervisor
-pep8
-httplib2
-flask
-pymongo
-zc.buildout
-psutil
-mysql-python
-argparse
-carbon
-pygments
-babel
-paste
-anyjson
-meld3
-# ssl (already included in standard library)
-cryptography
-py
-tornado
-pyopenssl
-greenlet
-kombu
-graphite-web
-docopt
-mako
-itsdangerous
-pillow
-wheel
-beautifulsoup4
-enum34
-pyflakes
-zope.interface
-decorator
-futures
-pastedeploy
-ordereddict
-setuptools-git
-fabric
-backports.ssl_match_hostname
-amqp
-numpy
-sphinx
-iso8601
-flake8
-celery
-pyparsing
-mccabe
-stevedore
-pytest
-webob
-gunicorn
-urllib3
-billiard
-jsonschema
-msgpack-python
-gevent
-logilab-common
-unittest2
+simplejson==3.10.0
+setuptools==34.3.2
+six==1.10.0
+requests==2.13.0
+virtualenv==15.1.0
+pip==9.0.1
+distribute==0.7.3
+python-dateutil==2.6.0
+certifi==2017.1.23
+boto==2.46.1
+pbr==2.0.0
+docutils==0.13.1
+pyasn1==0.2.3
+pyyaml==3.12
+jinja2==2.9.5
+markupsafe==1.0
+pytz==2016.10
+nose==1.3.7
+lxml==3.7.3
+pycrypto==2.6.1
+rsa==3.4.2
+colorama==0.3.7
+botocore==1.5.27
+cffi==1.10.0
+awscli==1.11.64
+coverage==4.3.4
+jmespath==0.9.2
+pycparser==2.17
+pika==0.10.0
+django==1.10.6
+psycopg2==2.7.1
+paramiko==2.1.2
+ecdsa==0.13
+sqlalchemy==1.1.6
+mock==2.0.0
+redis==2.10.5
+werkzeug==0.12.1
+selenium==3.3.1
+bcdoc==0.16.0
+supervisor==3.3.1
+pep8==1.7.0
+httplib2==0.10.3
+flask==0.12
+pymongo==3.4.0
+zc.buildout==2.9.2
+psutil==5.2.0
+mysql-python==1.2.5
+argparse==1.4.0
+carbon==0.9.15
+pygments==2.2.0
+babel==2.3.4
+paste==2.0.3
+anyjson==0.3.3
+meld3==1.0.2
+cryptography==1.8.1
+py==1.4.33
+tornado==4.4.2
+pyopenssl==16.2.0
+greenlet==0.4.12
+kombu==4.0.2
+graphite-web==0.9.15
+docopt==0.6.2
+mako==1.0.6
+itsdangerous==0.24
+pillow==4.0.0
+wheel==0.29.0
+beautifulsoup4==4.5.3
+enum34==1.1.6
+pyflakes==1.5.0
+zope.interface==4.3.3
+decorator==4.0.11
+futures==3.0.5
+pastedeploy==1.5.2
+ordereddict==1.1
+setuptools-git==1.2
+fabric==1.13.1
+backports.ssl_match_hostname==3.5.0.1
+amqp==2.1.4
+numpy==1.12.1
+sphinx==1.5.3
+iso8601==0.1.11
+flake8==3.3.0
+celery==4.0.2
+pyparsing==2.2.0
+mccabe==0.6.1
+stevedore==1.21.0
+pytest==3.0.7
+webob==1.7.2
+gunicorn==19.7.1
+urllib3==1.20
+billiard==3.5.0.2
+jsonschema==2.6.0
+msgpack-python==0.4.8
+gevent==1.2.1
+logilab-common==1.4.0
+unittest2==1.1.0
 prettytable
```

Comment:
Why did `prettytable` and `html5lib` not get versions?

Also, I'm going to sort these files once this is merged.